cmd/mdatagen/loader.go

```diff
@@ -201,11 +201,17 @@ func (a attribute) TestValue() string {
 	return ""
 }
 
+type goLeak struct {
+	Skip   bool     `mapstructure:"skip"`
+	Ignore []string `mapstructure:"ignore"`
```

Comment:
There are a few places in `contrib` that require using `goleak.IgnoreAnyFunction` instead of `goleak.IgnoreTopFunction`. We may need to create some kind of `Ignore` that has the string that needs ignored, and then type of ignore to differentiate between top/any, or other possible future required options.

Reply:
I've added an option for the following:

```

tests:
  goleak:
    ignore:
      top:
        - "go.opencensus.io/stats/view.(*worker).start"
      any:
        - "go.opencensus.io/stats/view.(*worker).somethingelse"
```